polyfill/lib/ecmascript.mjs

```diff
@@ -607,6 +607,16 @@ export const ES = ObjectAssign({}, ES2019, {
       GetSlot(dateTime, NANOSECOND)
     );
   },
+  GetOffsetNanosecondsFor: (timeZone, absolute) => {
+    const offsetNs = timeZone.getOffsetNanosecondsFor(absolute);
```

Comment:
Does this mean I have to update my PR once rebased on top of this to change this, and add new tests to handle `getOffsetNanosecondsFor = undefined`?

Reply:
Might be easier if I deal with it.